modules/indexing/src/test/java/org/apache/ignite/testsuites/IgniteBinaryCacheQueryTestSuite.java

```diff
@@ -585,7 +586,8 @@
     RowCountTableStatisticsSurvivesNodeRestartTest.class,
 
     SqlViewExporterSpiTest.class,
-    SystemViewSelfTest.class
+    SystemViewSelfTest.class,
+    QueryCursorSpliteratorCallsTest.class
```

Comment:
The list of suite's classes is divided into several named blocks marked with comments. Please, find the comment `// Queries tests.` and move this line to the end of that block.
Also, please pay attention to my today's comment on class QueryKeyValueSpliterator at line 48 about `@Override` annotation.

Reply:
Hej, one question - I started this feature branch from ignite-2.8.1 branch since it's a bug in existing release. Is it ok if I pull upstream/ignite-2.8.1 only?